pkg/tsdb/azuremonitor/azuremonitor_test.go

```diff
@@ -59,26 +60,28 @@ func TestNewInstanceSettings(t *testing.T) {
 	tests := []struct {
 		name          string
 		settings      backend.DataSourceInstanceSettings
-		expectedModel types.DatasourceInfo
+		expectedModel *types.DatasourceInfo
 		Err           require.ErrorAssertionFunc
+		setupContext  func(ctx context.Context) context.Context
 	}{
 		{
-			name: "creates an instance",
+			name: "current user authentication disabled by feature toggle",
```

Comment:
Nit: Did you intend to remove the original instance creation test? I think we should keep this as a JIC, wdyt?